Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -27,21 +27,27 @@ function Get-AgentIPRange
 
     $ErrorActionPreference = 'Continue'
 
-    ( Invoke-Expression "& '$sqlCmd' --% $sqlCmdArgs" -ErrorVariable errors -OutVariable output 2>&1 ) | Out-Null   
+    $output = ( Invoke-Expression "& '$sqlCmd' --% $sqlCmdArgs" -ErrorVariable errors 2>&1 ) | Out-String
 
     $ErrorActionPreference = 'Stop'
 
     if($errors.Count -gt 0)
```

Comment:
$ErrorActionPreference is continue.
Does that mean we can have more than 1 error. If yes, we will end up losing all the error messages except the first one.